[DOCS-12228] Replace sensitive data issues with findings (#32200)
DataDog/documentation: content/en/security/_index.md

[Sensitive Data Scanner][24] can help prevent sensitive data leaks and limit non-compliance risks by discovering, classifying, and optionally redacting sensitive data. It can scan for sensitive data in your telemetry data, such as application logs, APM spans, RUM events, and events from Event Management. It can also scan for sensitive information within your cloud storage resources. 

> After you [set up Sensitive Data Scanner][25], use the Findings page to see details of sensitive data findings that have been identified, so that you can triage, investigate, and remediate the findings.

Reviewer: ```suggestion
After you [set up Sensitive Data Scanner][25], use the Findings page to see details of detected sensitive data. From there, you can triage, investigate, and remediate the findings.
```
